scss/main/objects/_objects.date-picker.scss

```diff
@@ -47,6 +47,10 @@
     @include transition-property(opacity, transform);
     @include transition-duration(0.6s);
     @include transition-timing-function($ease-out-quint);
+
+    a {
+      cursor: pointer;
```

Comment:
Is this really necessary? An `a` should already have the pointer. And using an `a` is way to general.